tests/phpunit/tests/test-choose-lang-domain.php

```diff
@@ -92,6 +92,7 @@ public function go_to( $url ) {
 		unset( $GLOBALS['wp_query'], $GLOBALS['wp_the_query'] );
 
 		// insert Polylang specificity
+		unset( $GLOBALS['wp_actions']['wp_default_styles'] ); // So `PLL_Choose_Lang::set_language()` doesn't calls `wp_styles()`, same behavior as production environment.
```

Comment:
I suggest to use [`remove_all_actions()`](https://developer.wordpress.org/reference/functions/remove_all_actions/) instead. and to do this in test setup.

Reply:
Done in 7c5287b9!